src/pybind/mgr/cephadm/ssl_cert_utils.py

```diff
@@ -198,7 +198,7 @@ def generate_cert(
 
         builder = x509.CertificateBuilder()
         root_ca_name = x509.Name([
-            x509.NameAttribute(NameOID.COMMON_NAME, u'cephadm-root'),
+            x509.NameAttribute(NameOID.COMMON_NAME, f'cephadm-root-{self.cluster_fsid}'),
```

Comment:
This line will cause all the newly generate certificates not to be in sync with currently loaded root-ca. This could happen after upgrading from an old system where Root CA CN was `cephadm-root` to a new system where the Root CA has the format `cephadm-root-<fsid>` impacting all the running services once they get redeployed.  To fix that we need to get the subject from the already loaded `root_cert` object.  This commit fixes this issue: https://github.com/ceph/ceph/pull/62106/commits/d4c66428ee904f3cc1dc89960cb31a47f163e844